specs/TransportProtocols.md

```diff
@@ -117,16 +117,17 @@ Long Polling requires that the client poll the server for new messages. Unlike t
 
 Since there is such a long round-trip-time for messages, given that the client must issue a request before the server can transmit a message back, Long Polling responses contain batches of multiple messages. Also, in order to support browsers which do not support XHR2, which provides the ability to read binary data, there are two different modes for the polling transport.
 
-A Poll is established by sending an HTTP GET request to `[endpoint-base]/poll` with the following query string parameters
+A Poll is established by sending an HTTP GET request to `[endpoint-base]` with the following query string parameters
 
 * `connectionId` (Required) - The Connection ID of the destination connection.
-* `supportsBinary` (Optional: default `false`) - A boolean indicating if the client supports raw binary data in responses
 
-When messages are available, the server responds with a body in one of the two formats below (depending upon the value of `supportsBinary`). The response may be chunked, as per the chunked encoding part of the HTTP spec.
+Adding the `Accept` header with value `application/vnd.microsoft.aspnetcore.endpoint-messages.v1+binary` indicates if the client supports raw binary data in responses.
```

Comment:
It's not totally clear that the `Accept` header is required in both cases, even when doing text.